docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md

```diff
@@ -0,0 +1,45 @@
+# Transfer your app from `appc` CLI to `ti` CLI
```

Comment:
Use same title as incoming link title. Other than titles, "Appc CLI" should be sufficient.

Reply:
fixed